### PR TITLE
ci: Use travis_fold on Travis CI only

### PR DESCRIPTION
**Pull request by hebasto**

### CODE REVIEW

File: ci/test/03_before_install.sh [hunk 1]
```diff
@@ -6,15 +6,20 @@
...
-  echo "travis_fold:start:${CURRENT_FOLD_NAME}"
+  echo "${FOLD_COMMAND}start:${CURRENT_FOLD_NAME}"
 }
 
 END_FOLD () {
   RET=$?
-  echo "travis_fold:end:${CURRENT_FOLD_NAME}"
+  echo "${FOLD_COMMAND}end:${CURRENT_FOLD_NAME}"
   if [ $RET != 0 ]; then
     echo "${CURRENT_FOLD_NAME} failed with status code ${RET}"
```
Comment (MarcoFalke) [inline ci/test/03_before_install.sh:24]: Pretty sure this is dead code due to use of our `errexit`.

File: ci/test/03_before_install.sh [hunk 2]
```diff
@@ -6,15 +6,20 @@
 
 export LC_ALL=C.UTF-8
 
+FOLD_COMMAND=""
+if [[ ${TRAVIS} == "true" ]]; then
+  FOLD_COMMAND="travis_fold:"
+fi
```
Comment (MarcoFalke) [inline ci/test/03_before_install.sh:12]: Not sure if it is worth it to keep travis related code in our ci files, given that we move away from it.

**Review by maflcko — COMMENTED** [review]
review ACK dde5f4a1a57a6fcbbadb47e35b2b409eadac0a2f, but I am not sure if it is worth it to keep travis related code in our ci files at all, given that we move away from it.

**Comment by fanquake** [comment]
I agree with Marco, lets just finish up removing the remaining Travis related code. I assume that'll be done as part of, or just after #20467.

**Comment by MarcoFalke** [comment]
#20467 is only about the linters. `./ci/test` doesn't use travis anymore already.

File: ci/test/03_before_install.sh (hunk 2 above)
Comment (MarcoFalke) [reply ci/test/03_before_install.sh:12]: ./ci/test/ doesn't use travis anymore

**Comment by hebasto** [comment]
Agree. After linter moving from Travis, all Travis related code should be just removed.

**Comment by hebasto** [comment]
Closed in favor of #20691.

**Pull request closed without merging**